### PR TITLE
DOP-1421 - Update promo code view

### PR DESCRIPTION
**Pull request by ck-makingsense**
Set view for promo code 

image

### CODE REVIEW

File: src/customJs/tools/promo_code/PromoCodeViewer.tsx [hunk 1]
```diff
@@ -11,8 +11,16 @@ export const PromoCodeViewer: ViewerComponent<PromoCodeValues> = ({
...
+      style={{
+        textAlign: values.alignment,
+        backgroundColor: values.backgroundColor,
+      }}
+      role="container"
+    >
+      <span style={{ color: values.textColor, fontSize: values.fontSize }}>
+        {values.promo_code}
+      </span>
+    </div>
```
Comment (andresmoschini) [inline src/customJs/tools/promo_code/PromoCodeViewer.tsx:24]: Is this the same markup that is being rendered in MSEditor?

**Comment by andresmoschini** [comment]
👍

File: src/customJs/tools/promo_code/PromoCodeViewer.tsx (hunk 1 above)
Comment (ck-makingsense) [reply src/customJs/tools/promo_code/PromoCodeViewer.tsx:24]: almost, I added the font size

File: src/customJs/tools/promo_code/PromoCodeViewer.tsx [hunk 2]
```diff
@@ -11,8 +11,16 @@ export const PromoCodeViewer: ViewerComponent<PromoCodeValues> = ({
...
-      <pre>{JSON.stringify(values, undefined, 2)}</pre>
-    </code>
+    <div
+      style={{
+        textAlign: values.alignment,
+        backgroundColor: values.backgroundColor,
+      }}
+      role="container"
+    >
+      <span style={{ color: values.textColor, fontSize: values.fontSize }}>
```
Comment (andresmoschini) [inline src/customJs/tools/promo_code/PromoCodeViewer.tsx:21]: At this stage, `backgroundColor`, `textColor` and `fontSize` are undefined, right?

Does it generate a valid HTML when these values are undefined?

Comment (ck-makingsense) [reply src/customJs/tools/promo_code/PromoCodeViewer.tsx:21]: that right is a valid HTML without style, at the next PR I will add the property to show the styles on the view

**Pull request closed without merging**